relay-general/src/store/normalize.rs

```diff
@@ -34,6 +37,18 @@ mod stacktrace;
 
 mod user_agent;
 
+/// Configuration for measurements normalization.
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(default, rename_all = "camelCase")]
+pub struct MeasurementsConfig {
+    /// A list of measurements that are built-in and are not subject to custom measurement limits.
+    #[serde(default, skip_serializing_if = "BTreeSet::<String>::is_empty")]
+    known_measurements: BTreeSet<String>,
+
+    /// The maximum number of measurements allowed per event that are not known measurements.
+    max_custom_measurements: usize,
```

Comment:
Not related to the diff -- if a project config sent by sentry contains a negative integer in this field, how does the config deserialization work? Do we throw an error for a bad project config?

Reply:
Yep, project config parsing will fail.